xml/System.Runtime.CompilerServices/SwitchExpressionException.xml

```diff
@@ -18,7 +18,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>Indicates that a switch expression that was non-exhaustive failed to match its input at runtime, e.g. in the C# 8 expression <code>3 switch { 4 =&gt; 5 }</code>.
+    <summary>Indicates that a switch expression that was non-exhaustive failed to match its input at runtime. For example, in the C# 8 expression <code>3 switch { 4 =&gt; 5 }</code>.
```

Comment:
This e.g. => For example change is strange to me... the former made sense grammatically, whereas the latter now has an incomplete sentence.

Reply:
@mairaw should we keep the "e.g."? Also, do we support `<code></code>` tags in Docs summaries? First time I see them.

Reply:
I'll put the example in the remarks.

Reply:
Done. Let me know if you prefer to have the example shown in a different way.